### PR TITLE
Adding note: How to file issues against this repo.

### PR DESCRIPTION
**Pull request by marstr**
Figured it is good to get this kinda thing in ASAP, even if the rest of the README still needs more work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,11 @@
 
+# Issues
+If you're reading this, you've likely noticed that this repository hasn't enabled issues. That isn't because we don't want your feedback! **Please file issues for the repository in one of the following repositories** as appropriate:
+
+  - [Azure/azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go/issues)
+  - [Azure/autorest](https://github.com/Azure/autorest)
+  - [Azure/azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs)
```
Comment (mcardosos) [inline README.md]: go-autorest is missing.
Maybe we should also add which repo to choose depending on the issue.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,10 @@
 # Issues
 If you're reading this, you've likely noticed that this repository hasn't enabled issues. That isn't because we don't want your feedback! **Please file issues for the repository in one of the following repositories** as appropriate:
 
-  - [Azure/azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go/issues)
-  - [Azure/autorest](https://github.com/Azure/autorest)
-  - [Azure/azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs)
+  - [Azure/azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go/issues) : When you're experiencing trouble with the Go SDK, but not with other languages, this is the repository to report bugs to. i.e. What we generate for Long Running Operations doesn't make sense or there's something hard coded that shouldn't be.
```
Comment (mcardosos) [inline README.md]: "Azure SDKs for other languages"

File: README.md [hunk 3]
```diff
@@ -2,9 +2,10 @@
...
+  - [Azure/go-autorest](https://github.com/Azure/go-autorest) : File bugs here when your troubles aren't related to a particular operation, or even the pattern of generation, but the core infrastructure that we use to build HTTP request or authenticate. i.e. Retry or Authentication aren't working the way you wanted.
+  - [Azure/autorest](https://github.com/Azure/autorest) : Should you run into duplicated types, or fundamentally flawed enums, and it seems consistent across languages, but doesn't seem like the problems lies with the actual Azure Service, the problem could lie with how we're modeling the problem as we see it in the Open API Spec. File bugs matching this description to our parent project, "Autorest".
```
Comment (mcardosos) [inline README.md]: I would make clear that this has to do with ARM generated code

**Comment by olydis** [comment]
# publish job

**Pull request merged**